tasks/ubuntu.task/os_boot.erb

```diff
@@ -4,6 +4,11 @@ exec >> /var/log/razor.log 2>&1
 
 <%= render_template("set_hostname") %>
 
+# ubuntu 18.04 networking doesn't wake up right away, so...
```

Comment:
After more research, this still looks like the most straightforward solution. I do think it should move above the `set_hostname` template above, since the template reaches out to Razor server if something fails.

Reply:
Done.